src/Console/Database/FixHtmlEncodingCommand.php

```diff
@@ -298,4 +333,59 @@ private function doubleEncoding(string $input): string
         ];
         return preg_replace($pattern, $replace, $input);
     }
+
+    /**
+     * Fix double encoded HTML entities in old followups generated from Outlook
```

Comment:
This double encoding was made by us.
```suggestion
     * Fix double encoded HTML entities in old followups
```